doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -848,6 +848,15 @@ Other libraries
 
 * Removed the following unused SDFW services: ``echo_service``, ``reset_evt_service``, and ``sdfw_update_service``.
 
+Common Application Framework (CAF)
+----------------------------------
```

Comment:
We have this section on top - line:no 714. We could move the whole section there.

Reply:
Oh, I was looking for "CAF", but the acronym is not there. I moved it now.